Sources/CrowdNotifierBaseSDK/QRCodeParser/QRCodeParser.swift

```diff
@@ -125,8 +125,8 @@ class QRCodeParser {
                              address: content.location,
                              notificationKey: content.notificationKey,
                              publicKey: entry.masterPublicKey,
-                             nonce1: entry.entryProof.nonce1,
-                             nonce2: entry.entryProof.nonce2,
+                             noncePreId: entry.entryProof.nonce1,
+                             nonceTimekey: entry.entryProof.nonce2,
```

Comment:
This has me a little worried. We're reinterpreting a V2 QR code into a V3 description. I can see how that can work, sort of, but I thought we would use the V2 identity computations for V2 codes?